src/config/layers.js

```diff
@@ -1358,6 +1371,46 @@ export const zweitausbildungRoutes = new Layer({
   ],
 });
 
+export const tarifverbundkarteLayer = new TarifverbundkarteLayer({
+  mapboxLayer: tarifverbundkarteDataLayer,
+  visible: true,
+  properties: {
+    hideInLegend: true,
+    popupComponent: 'TarifverbundkartePopup',
+  },
+  styleLayers: [
+    {
+      id: 'verbundskarte',
+      source: 'tarifverbundkarte',
+      'source-layer': 'ch.sbb.tarifverbundkarte',
+      type: 'fill',
+      paint: {
+        'fill-color': '#627BC1',
+        'fill-outline-color': '#627BC1',
+        'fill-opacity': 0,
+      },
+    },
+    {
+      id: 'verbundskarte.zpass',
+      source: 'tarifverbundkarte',
+      'source-layer': 'ch.sbb.tarifverbundkarte.zpass',
+      type: 'fill',
+      paint: {
+        'fill-opacity': 0,
+      },
+    },
+    {
+      id: 'verbundskarte.zonen',
+      source: 'tarifverbundkarte',
+      'source-layer': 'ch.sbb.tarifverbundkarte.zonen',
+      type: 'fill',
+      paint: {
+        'fill-opacity': 0,
+      },
+    },
+  ],
```

Comment:
Add this style layers directy in the style

Reply:
Done